wazo_setupd/plugins/setup/services.py

```diff
@@ -110,6 +110,13 @@ def post_confd_wizard(self, language, password, license_accepted):
 
         discover = c.wizard.discover()
 
+        if len(discover['nameservers']) > 2:
```

Comment:
3 is also valid, this limit comes from resolv.conf

Reply:
I also don't know what's wrong with a 400 since the validation happens in setupd.

Reply:
You can read my explaination in the [JIRA](https://wazo-dev.atlassian.net/browse/WAZO-1496) ticket
400 implies that you can resolve it by changing your request, which is wrong, you cannot. The only way to resolve it is modify your server, because `nameservers` value come from your server, not from the request.

Reply:
> You can read my explaination in the [JIRA](https://wazo-dev.atlassian.net/browse/WAZO-1496) ticket
> 400 implies that you can resolve it by changing your request, which is wrong, you cannot. The only way to resolve it is modify your server, because `nameservers` value come from your server, not from the request.
> 
> > 3 is also valid, this limit comes from resolv.conf
> 
> I don't get your point? It's exactly the reason why wazo's wizard doesn't support more than 2 nameservers

recolv.conf, sysconfd and wazo-confd all accept 3 nameservers